### ISSUE TITLE
License

### ISSUE THREAD

**Issue by pascalberger** [gerlocian/ng-html5dragdrop#3]
First of all thanks for this module! I saw that you put it under GPL license. Now this being a JavaScript library this means that at least things get very complicated, if it doesn't mean that this library cannot be used. Problems being if minification is allowed, there are even discussions if it is legal to run JavaScript in browsers under a license not compatible with the GPL, etc.

Is there a reason why you have chosen GPL over a more suitable license for JavaScript, like eg. MIT license?

**Comment by gerlocian** [comment]
Honestly, my intention was to allow anyone to use, distribute and edit the source so long as I had credit for the original. The problem is I'm not a lawyer, so finding the license that was right for what I was trying to do was difficult. In the end I chose GPL because I thought it would guarantee those privileges.

After looking into it more and reading up on the differences, I believe that MIT is the correct license. As such, I've updated the repository to version 1.1.0, with the new minor version upgrade signifying when the license changed from GPL to MIT. If you get the 1.1.0 version, this will be with the MIT license.
